api/tests/core/finance/test_api.py

```diff
@@ -232,9 +232,11 @@ def test_price_booking_that_has_been_unused_and_then_used_again(self):
         assert pricing.status == models.PricingStatus.VALIDATED
 
     def test_price_booking_that_is_not_used(self):
-        booking = bookings_factories.BookingFactory()
-        pricing = api.price_booking(booking)
-        assert pricing is None
+        bookings_factories.BookingFactory()
+        try:
+            api.price_bookings(datetime.datetime(1970, 1, 1), 100)
+        except Exception as err:
+            raise pytest.fail(f"DID RAISE {err}")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Ce test me laisse perplexe :sweat_smile: 
Pourquoi on catch pour raise ?
Et que représente `100` ici ?

Reply:
Pour que ça soit explicite que le test casse en cas d'exception quelconque raised quelque part, sinon la définition du test est très vide ^^"

```python

    def test_price_booking_that_is_not_used(self):
        bookings_factories.BookingFactory()
        api.price_bookings(datetime.datetime(1970, 1, 1), 100)
```

100 c'est le batch_size @lixxday

Reply:
Si la fonction raise, le test va casser, je pense que c'est pas la peine d'attraper l'exception seulement pour la jeter de nouveau

Proposition: `api.price_bookings(datetime.datetime(1970, 1, 1), batch_size=100)` pour aider le lecteur :smile: